### DISCUSSION TITLE
Order by the IN parameters

### DISCUSSION THREAD

**Discussion by dstoyanoff** [prisma/prisma#19976] [Q&A] [answered]
Hello everyone,
I have a case where I want to do a query on ids using IN, but want to guarantee the order of resulted items would be the same as the order of the IN ids.

Is this currently possible with prisma? I found this stack overflow thread in regards to the postgres functionality: https://stackoverflow.com/questions/866465/order-by-the-in-value-list

**Comment by nurul3101** [comment, marked as answer]
Hello @dstoyanoff 👋 

Ordering by IN parameters isn't natively supported by Prisma, you would need to use [Raw Queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) to achieve this.

**Reply by dstoyanoff** [reply]
Thank you. Is there any particular reason why it won't be available, just out of curiosity? Thanks

**Reply by nurul3101** [reply]
Oh, I meant to say that Ordering by IN parameters isn't supported by Prisma yet. I didn't mean to say that it won't ever be supported. 🙂 
For now, you would need to use Raw Queries.
